utils/build-script-utils/src/lib.rs

```diff
@@ -17,8 +17,8 @@
 
 //! Crate with utility functions for `build.rs` scripts.
 
-mod version;
 mod git;
+mod version;
```

Comment:
It might be preferable to avoid the re-ordering when introducing a rustfmt toml to minimize the changeset.

Reply:
better to have one hit of pain/disruption than several waves of disruption. As someone else pointed out the diff will be XXL either way.

Reply:
> better to have one hit of pain/disruption than several waves of disruption. As someone else pointed out the diff will be XXL either way.


Yeah fair.